### DISCUSSION TITLE
Proposal: Move community meetings from Google Meet to Zoom

### DISCUSSION THREAD

**Discussion by leecalcote** [meshery/meshery#1110] [General] [answered]
Moving to Zoom will solve these current pain points with Google Meet:
- Google Meet's behavior is such that it requires approval for admittance of attendees.
- Only the Google Meet Enterprise plan allows for livestreaming to Youtube.

Switching to Zoom from Google Meet may have unintended consequences, though. Like the integration we have configured with Youtube and Twitter to automatically send a tweet when a new community call is published. Is there general support for moving all calls Zoom?

**Comment by kushthedude** [comment, marked as answer]
+1 from Zoom.

**Reply by kushthedude** [reply]
@leecalcote We can raise this topic in this week's community meeting, thoughts ?
